keps/sig-node/2400-node-swap/README.md

```diff
@@ -875,8 +998,18 @@ with and without the feature, are necessary. At the very least, think about
 conversion tests if API types are being modified.
 -->
 
-N/A. This should be tested separately for scenarios with the flag enabled and
-disabled.
+There are extensive tests to ensure that the swap feature as expected.
+
+Unit tests are in place to test that this feature operates as expected with
```

Comment:
I thought we limited memory swap to cgroupv2 only?

Reply:
Yes, you're correct.

In cgroup v1 we always set swap limit to 0. IOW swap is never accessible to workloads under v1.
What I meant here is that the tests validate this behavior.